core-bundle/src/EventListener/Security/SwitchUserListener.php

```diff
@@ -47,9 +46,6 @@ public function __invoke(SwitchUserEvent $event): void
         $sourceUser = $token->getUserIdentifier();
         $targetUser = $event->getTargetUser()->getUserIdentifier();
 
-        $this->logger->info(
-            sprintf('User "%s" has switched to user "%s"', $sourceUser, $targetUser),
-            ['contao' => new ContaoContext(__METHOD__, ContaoContext::ACCESS, $sourceUser)]
-        );
+        $this->logger->info(sprintf('User "%s" has switched to user "%s"', $sourceUser, $targetUser));
```

Comment:
Monolog/PSR supports placeholders for the message, should we use that? https://github.com/Seldaek/monolog/blob/main/doc/message-structure.md#log-message-structure
we would need to make sure the placeholders are replaced for the backend log message, but I think that's already the case.

Reply:
This would be nice. However, this is out of the scope of this pull request in my opinion and may be added at a later point in time.